doc/design/vic-machine.md

```diff
@@ -63,6 +63,111 @@ To create a VCH, disambiguation of which resource to use from the available set
   - volumes
 
 
+## Usage Examples
+
+These are various examples intended to illustrate how vic-machine is intended to be used. These are illustrations rather than the finalized naming so if something doesn't make sense ask about it. 
+
+### Fully specified command line
+
+```
+vic-machine create
+-TO=https://root@****:vcenter/
+-name=edw-web
+-compute-resource=/example-datacenter/host/mycluster/Resources/edw-pool/web
+-image-store=ds://datastore/containers/edw
+-container-store=ds://datastore/edw/edw/web
+-client-network="VM Network"
+-management-network="management-net"
+-volume-store=ds://edw-san/reports:reports
+-volume-store=ds://edw-san/data:data
+-docker-network=private-vlan12d7f2:bridge
+-docker-network=edw-corp-net:backend
+-docker-network-ipam=edw-corp-net:10.118.78.128-192/24,10.118.78.1
+-container=oracledb.edw.corp.net:db.backend
```

Comment:
So the container created in db.backend network, should have domain oracledb.edw.corp.net?